rust/repo/{{ cookiecutter.repo_name }}/deny.toml

```
@@ -19,6 +19,14 @@ allow = [
     "ISC",
     "MIT",
     "Unicode-DFS-2016",
+{%- if cookiecutter.license == "GPL-3.0-only" or cookiecutter.license == "AGPL-3.0-only" -%}
+    "GPL-3.0-only",
+    "GPL-3.0-or-later",
+{%- endif -%}
+{%- if cookiecutter.license == "AGPL-3.0-only" -%}
+    "AGPL-3.0-only",
+    "AGPL-3.0-or-later",
+{$- endif -%}
```

:x: Couple of things
1. Incorrect syntax, a `$` was used instead of `%`
2. similar to other template expressions, suggest omitting trailing `-` to keep ending array brace on next line
3. 
```suggestion
{%- endif %}
```